kci_data

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2018, 2019 Collabora Limited
+# Author: Michal Galka <michal.galka@collabora.com>
+#
+# This module is free software; you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the Free
+# Software Foundation; either version 2.1 of the License, or (at your option)
+# any later version.
+#
+# This library is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this library; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+
+import sys
+
+import kernelci.config.data
+from kernelci.data import submit
+from kernelci.cli import Args, Command, parse_args
+
+
+class cmd_list_configs(Command):
+    help = "list all data endpoint names"
+
+    def __call__(self, config_data, *args, **kwargs):
+        db_configs = config_data['db_configs']
+        for config_name in db_configs:
+            print(config_name)
+        return True
+
+
+class cmd_submit(Command):
+    help = "Submit data to specified storage"
```

Comment:
```suggestion
    help = "Submit data to the specified database"
```

Reply:
"storage" sounds more like files or objects, we're dealing with databases here.

Reply:
Good point. corrected